components/SpeakersSection.js

```diff
@@ -41,6 +61,19 @@ export default function TicketsSection () {
           display: block;
           margin: 50px 0 0;
         }
+        @media screen and (max-width: 991px) {
+          .container {
+            width: 750px;
+          }
+        }
+        @media screen and (max-width: 768px) {
+          .container {
+            width: 100%;
```

Comment:
If we change the above (line 34) to `max-width`, then we don’t need this as well.